modules/ROOT/pages/migration-security-filters.adoc

```diff
@@ -3,16 +3,16 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-In Mule 3, the Security Filters where part of the Mule Module Security. In Mule 4, this where migrated to the xref:connectors::validation/validation-connector.adoc[Mule Validation Module].
+In Mule runtime engine (Mule) 3, the Security Filters are part of the Mule Module Security. In Mule 4, security filters are in the xref:connectors::validation/validation-connector.adoc[Mule Validation Module].
```

Comment:
In one case Security Filters is capped, then later it's not. I think it doesn't need caps.

Reply:
fixed.